appchain-anchor/src/lookup_array.rs

```diff
@@ -118,18 +118,31 @@ where
         self.end_index = *index;
     }
     ///
-    pub fn clear(&mut self) {
-        for index in self.start_index..self.end_index + 1 {
+    pub fn clear(&mut self) -> MultiTxsOperationProcessingResult {
```

Comment:
I have same suggestions as `appchain_messages's clear`.

Reply:
Will change to compare with `end_index`.